modules/google_service_account/README.md

```diff
@@ -1,5 +1,14 @@
 # Google Service Account Terraform Component Module
 
+Module for managing Google Service Accounts plus:
+* Service Account Keys
+* IAM bindings for projects
+* IAM bindings for project folders
```

Comment:
```suggestion
* IAM bindings for folders
```

Reply:
I assume we're not differentiating between folders that contain folders, and folders that contain only projects? Or is the latter what we mean by "project folders"?

Reply:
Wanted to be explicit what kind of folders they are for, since folders are a concept in GCS also.

The actual API for this is `resourcemanager` which isn't obvious that it's relating to GCP projects and makes the wording pretty weird in most cases.